src/test/TEST_examples/gpu.saved.cuda

```diff
@@ -0,0 +1,72 @@
+# Output file: gpu.out.1
```

Comment:
@rfalgout Do I name this file correctly (.saved.cuda) for `machine-lassen`? Not sure if the `save` mechanism of this test (with `-eo`) is treated the same as the others (with `-ro`). Thanks!

Reply:
The mechanism is the same for '-ro:' and '-eo:'.  You need to use the '-rt' flag to pass additional options through to the `runtest.sh` script.  So, it looks like you need `-eo: -gpu -rt -mpibind -save cuda`.